design/one-pager-definition-of-done.md

```diff
@@ -0,0 +1,46 @@
+# Definition of done
+
+* Owner: Daniel Suskin (@suskin)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Introduction
+
+*Why does this document exist?*
+
+The goal of this document is to answer the questions:
+
+* As a developer, when should I consider a chunk of work to be complete?
+* As a reviewer, what should I be looking for to verify that a chunk of work is
+  complete?
+
+## Definition of work
+
+Work means a contribution to the project, typically code.
+
+## Definition of done
+
+The following criteria **MUST** be met for work to be considered done:
+
+* Code is reviewed and approved
+* Code passes linting (use `make reviewable` to lint)
+* All error conditions are reported into the log or as an event, as
+  appropriate
+* Code is tested
```

Comment:
Is this purposefully vague about how a code has been tested? If so, should we suggest a unit testing goal or attempt to briefly summarise our philosophy toward unit tests in the below section?

Reply:
I have added some more detail to try to make things clearer